src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -662,15 +662,6 @@ private static bool IsChunkedModeForSend(HttpRequestMessage requestMessage)
             // Serialize entity-body (content) headers.
             if (requestMessage.Content != null)
             {
-                // TODO (#5523): Content-Length header isn't getting correctly placed using ToString()
```

Comment:
We can't remove this workaround.  WinHttpHandler binary builds for a separate NuGet package that is also used on .NET Framework.  And .NET Framework would not have this fix for Content-Length header.

Reply:
So can it just be conditioned as `#if netfx`?

Reply:
We don't build the WinHttpHandler package for NETFX.  We built it for NETSTANDARD only right now.  And NETSTANDARD means that the binary can be used for either a NetCore app or a .NET Framework app.

So, I'm not sure you can remove this code at all because there isn't a clear way at build time to distinguish the different runtime environments.  While it might be possible to do a runtime detect to see if you're running on .NET Framework vs. .NET Core, I don't think it is worth the code to check that.

Reply:
That said, I do see the various build and configuration targets in this file:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http.WinHttpHandler/src/Configurations.props

You can try doing a ifdef of netfx around the code since that define is used in the CSPROJ file:
https://github.com/dotnet/corefx/blob/f7b92f4d486f9516cc08596193e11b4601745a1e/src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.csproj#L15

But you'll need to test that change to make sure it behaves as expected.  It might be better to leave this code in for now.

Reply:
OK. Thank you.